gp-includes/things/original.php

```diff
@@ -280,16 +280,33 @@ public function import_for_project( $project, $translations ) {
 			if ( $close_original ) {
 				$original = $originals_by_key[ $close_original ];
 
+				/**
+				 * Filters whether to set existing translations to fuzzy.
+				 *
+				 * This filter is called when a new  string closely match an existing possibly dropped string.
+				 *
+				 * @since 2.3.0
+				 *
+				 * @param bool   $do_fuzzy Whether to set existing translations to fuzzy. Default true.
+				 * @param object $data     The new original data.
+				 * @param object $original The previous original being replaced.
+				 */
+				$do_fuzzy = apply_filters( 'gp_close_original_fuzzy_translations', true, (object) $data, $original );
```

Comment:
A better name for the filter and this looks ready to go. Perhaps something related to `set_translations_for_original_to_fuzzy()`